charts/Controls.tsx

```diff
@@ -442,8 +443,8 @@ class TimelineControl extends React.Component<{ chart: ChartConfig }> {
                 <Timeline
```

Comment:
Awesome generalization. Maybe at some point we could consolidate these almost identical Timeline elements into 1 and only change the different properties in each switch.